src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -72,7 +73,28 @@ const std::map<std::string, std::string> GetModuleTypeToDataMapping() {
   return mapping;
 }
 
-const std::map<std::string, std::string> GetModuleDataToCapabilitiesMapping() {
+std::map<std::string, std::string> GetLightCapabilitiesMapping() {
+  std::map<std::string, std::string> mapping = {
+      {message_params::kId, strings::kName},
+      {message_params::kStatus, strings::kStatusAvailable},
+      {message_params::kDensity, strings::kDensityAvailable},
+      {message_params::kSRGBColor, strings::kSRGBColorSpaceAvailable}};
+  return mapping;
+}
+
+std::map<std::string, std::string> GetModuleDataCapabilitiesMapping() {
+  std::map<std::string, std::string> mapping = {
+      {message_params::kRadioControlData, strings::kradioControlCapabilities},
+      {message_params::kClimateControlData,
+       strings::kclimateControlCapabilities},
+      {message_params::kAudioControlData, strings::kaudioControlCapabilities},
+      {message_params::kLightControlData, strings::klightControlCapabilities},
+      {message_params::kHmiSettingsControlData,
+       strings::khmiSettingsControlCapabilities}};
```

Comment:
Seat control capabilities?

Reply:
SeatControl functionality will be in PR related to rc seat